R/multitables.R

```diff
@@ -85,25 +85,32 @@ setMethod("is.public<-", "Multitable", function (x, value) {
 #' name(m) # [1] "gender + age4 + marstat"
 #' }
 #' @export
-newMultitable <- function (formula, data, name, ...) {
-    ## Validate inputs
-    if (missing(formula)) {
-        halt("Must provide a formula")
-    }
-    if (missing(data) || !is.dataset(data)) {
-        halt(dQuote("data"), " must be a Dataset")
-    }
+newMultitable <- function (formula, data, name, multitable=NULL, ...) {
+
+    if (missing(multitable)){
```

Comment:
I also would want an assignment method that would let me do e.g. `multitables(ds)[["Copied mult"]] <- multitables(other_ds)[["mult"]]` that calls importMultitable, but I won't make you learn all of the R sorcery required to do that.